### ISSUE TITLE
Get production deployment ready

### ISSUE THREAD

**Issue by j08lue** [NASA-IMPACT/veda-config-ghg#72]
It seems like we already have a pipeline for deployment to production (S3)

https://github.com/NASA-IMPACT/veda-config-ghg/blob/71b31da5f740a9b0b252524ca41af7bb0500962a/.github/workflows/deploy.yml#L123

However, it is unclear to me what is currently deployed at https://ghg.center - some older version? From which branch / tag / commit?

We need to get the pipeline in shape and the production deployment process documented, please, so we can get a deployment up on https://ghg.center on 6th of September, if possible.

## Acceptance criteria

- [ ] Latest version deployed to https://ghg.center
- [ ] Process documented and ready for 6th of September

**Comment by j08lue** [comment]
@slesaad, how does production deployment currently happen? It looks like the [deploy pipeline](https://github.com/NASA-IMPACT/veda-config-ghg/blob/71b31da5f740a9b0b252524ca41af7bb0500962a/.github/workflows/deploy.yml) on the `veda-config-ghg` repo is set up to deploy to production from `main`, but there is no `main` branch?

We just need to make sure we have a working and documented mechanism.

**Comment by slesaad** [comment]
The production deployment should happen on push to main branch automatically. When we're ready to deploy, all we need to do is create the main branch from the develop branch and that's it.

What we have in ghg.center right now is just me manually deploying to see everything works correctly, the merge to main should overwrite that.

**Comment by j08lue** [comment]
I created `main` off current `develop` for testing the process ahead of the big day tomorrow. 

Forgot that the result would not have password protection 😬 so we need to move a bit faster on that:

- #73 

Looks like the page is not loading on https://ghg.center, though, due to CORS errors.

**Comment by slesaad** [comment]
>Looks like the page is not loading on https://ghg.center/, though, due to CORS errors trying to load assets from https://staging.ghg.center/ for some reason.

I looked into this a bit, and it was apparently because to branched off of the `develop` branch rather than merged into it. Because we cache using the git-sha, the deployment was restoring the cache from when it was built before for staging.ghg.center because the git-sha was same for the new `main` branch.

I pushed a small change to `main` and it deployed and loaded as expected. Should work fine on merge now.

I've deleted the files from the production s3 bucket since it's not password-protected yet.
